vllm/core/block/common.py

```diff
@@ -7,7 +7,19 @@
 RefCount = int
 
 
-class RefCounter:
+class RefCounterInterface:
```

Comment:
Maybe you can use `typing.Protocol` for this.

Reply:
Yeah good point. It is because we can have different APIs for read-only vs not right?

Reply:
I just thought that `typing.Protocol` is the standard way of declaring interfaces in Python